PyPoE/cli/exporter/wiki/parsers/skill.py

```diff
@@ -742,7 +742,7 @@ def export(self, parsed_args, skills):
             if not parsed_args.allow_skill_gems and skill in \
                     self.rr['SkillGems.dat'].index['GrantedEffectsKey']:
                 console(
-                    'Skipping skill gem skill "%s"' % skill['Id'],
+                    'Skipping skill gem skill "{}" at row {}'.format(skill['Id'], skill.rowid),
```

Comment:
https://github.com/Project-Path-of-Exile-Wiki/PyPoE/pull/37#discussion_r744102703

Reply:
Fixed